desktop-widgets/command_edit.cpp

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include "command_edit.h"
+#include "core/divelist.h"
+
+namespace Command {
+
+template<typename T>
+EditBase<T>::EditBase(const QVector<dive *> &divesIn, T newValue, T oldValue) :
+	value(std::move(newValue)),
```

Comment:
I looked ahead and it seems we only ever instantiate this for QString. So why is this a template class?
Or am I missing something?
 (OK, that DOES seem rather likely)

Yeah, coming back to state that I found this in all of the following commits... :-/